services/fxa/whitelist.yml

```diff
@@ -3,115 +3,116 @@ payments:
     - 54.68.203.164
     - 35.164.199.47
     - 52.36.241.207
-  stage:
-    - 3.217.6.148
-    - 3.214.25.122
-    - 3.94.151.18
-  qa:
-    - 3.217.6.148
-    - 3.214.25.122
-    - 3.94.151.18
+#  stage:
```

Comment:
why commented out instead of delete? `#` are for comments only, not dead code

Reply:
addressed in update